packages/trpc/server/routers/viewer/eventTypes/update.handler.ts

```diff
@@ -241,13 +241,13 @@ export const updateHandler = async ({ ctx, input }: UpdateOptions) => {
     }
   }
 
-  if (input?.price || input.metadata?.apps?.stripe?.price) {
-    data.price = input.price || input.metadata?.apps?.stripe?.price;
+  if (input.metadata?.apps?.stripe?.price) {
+    data.price = input.metadata?.apps?.stripe?.price;
     const paymentCredential = await ctx.prisma.credential.findFirst({
       where: {
         userId: ctx.user.id,
         type: {
```

Comment:
Should we used appId instead? since type is deprecated.

https://github.com/calcom/cal.com/blob/main/packages/prisma/schema.prisma#L128-L129